media-source-respec.html

```diff
@@ -653,29 +653,30 @@
             <dd>
               <ol>
                 <li>Set the media element's <a def-id="delaying-the-load-event-flag"></a> to false.</li>
                 <li>Set the <a def-id="readyState"></a> attribute to <a def-id="open"></a>.</li>
                 <li>
                   <a def-id="Queue-a-task-to-fire-an-event-named"></a> <a def-id="sourceopen"></a> at the <a>MediaSource</a>.</li>
-                <li>Continue the <a def-id="resource-fetch-algorithm"></a> by running the <a def-id="perform-potentially-cors-enabled-fetch"></a> step, with these clarifications:
+                <li>Continue the <a def-id="resource-fetch-algorithm"></a> by running the remaining <a def-id="Otherwise-mode-is-local"></a> steps, with these clarifications:
                   <ol>
-                    <li>Text in the <a def-id="resource-fetch-algorithm"></a> that refers to "the download" or "bytes received" refers to data passed in via <a def-id="appendBuffer"></a>.</li>
-                    <li>References to HTTP in the <a def-id="resource-fetch-algorithm"></a> do not apply because the HTMLMediaElement does not fetch media data via HTTP when a <a>MediaSource</a> is attached.</li>
-                    <li>Text in the <a def-id="resource-fetch-algorithm"></a> that refers to transitions to "NETWORK_IDLE" and firing "suspend" events, and the paragraph that begins with "User agents may decide to not download more content at any time", do not apply to a media element that has a <a>MediaSource</a> object attached by this algorithm.</li>
+                    <li>Text in the <a def-id="resource-fetch-algorithm"></a> or the <a def-id="media-data-processing-steps-list"></a> that refers to "the download", "bytes received", or "whenever new data for the current media resource becomes available" refers to data passed in via <a def-id="appendBuffer"></a>.</li>
+                    <li>References to HTTP in the <a def-id="resource-fetch-algorithm"></a> and the <a def-id="media-data-processing-steps-list"></a> do not apply because the HTMLMediaElement does not fetch media data via HTTP when a <a>MediaSource</a> is attached.</li>
                   </ol>
                 </li>
               </ol>
             </dd>
           </dl>
+          <p class="note">An attached MediaSource does not use the remote mode steps in the <a def-id="resource-fetch-algorithm"></a>, so the media element will not fire "suspend" events. User agents MAY still fire "stalled" events as these [[HTML51]] references changed after implementations of this specification stabilized.</p>
+          <p class="note">When a MediaSource is attached via <code>src</code> attribute or selected child <code>&lt;source&gt;</code>'s <code>src</code> attribute set to a <a def-id="MediaSource-object-URL"></a>, then the <code>srcObject</code> attribute of the HTMLMediaElement MAY not reflect this as these [[HTML51]] references changed after implementations of this specification stabilized.</p>
```

Comment:
Oh. That's not quite how either mwatson2@ (in https://github.com/w3c/media-source/issues/99#issuecomment-234029419) or I understood this, but rereading the current whatwg spec (https://html.spec.whatwg.org/multipage/embedded-content.html#dom-media-srcobject) shows that assigned media provider object isn't changed by the resource fetch algorithm. Could it be? The idea is that it would rock if devtools users could query a media element for its srcObject to aid debugging of MediaSource (or MediaStream, etc) without modifying the original page to expose the MediaSource object some other way.